### ISSUE TITLE
is normalization step needed in feature preparation?

### ISSUE THREAD

**Issue by donganlei** [yinlou/mltk#23]
If the original features are at different ranges other than [-1, 1], do we need to normalize/calibrate their values before running mltk.predictor.gam.GAMLearner? Or, GAMLearner will take care of it?

Another question is: for mltk.predictor.evaluation.Evaluator, does can we find its metric output? I don't see the metric numbers displayed in output display or saved in any output file?

**Comment by yinlou** [comment]
No, you don't need to do any special preprocessing for features. Features of different scales are fine for GAM and GA2M.

You can find Evaluator's documentation here: https://github.com/yinlou/mltk/wiki/Model-Selection%2C-Evaluation-and-Prediction#model-evaluation
